src/Services/InsuranceService.php

```diff
@@ -14,13 +14,36 @@
 
 namespace OpenEMR\Services;
 
+use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Validators\ProcessingResult;
+use OpenEMR\Services\AddressService;
+use OpenEMR\Validators\CoverageValidator;
 use Particle\Validator\Validator;
 
-class InsuranceService
+class InsuranceService extends BaseService
 {
 
+    private const COVERAGE_TABLE = "insurance_data";
+    private const PATIENT_TABLE = "patient_data";
+    private const INSURANCE_TABLE = "insurance_companies";
+    private $uuidRegistry;
+    private $coverageValidator;
+    private $addressService = null;
+
+
+    /**
+     * Default constructor.
+     */
     public function __construct()
     {
+        $this->addressService = new AddressService();
+        $this->uuidRegistry = new UuidRegistry(['table_name' => self::COVERAGE_TABLE]);
+        $this->uuidRegistry->createMissingUuids();
```

Comment:
could also change above 2 lines to:
```
(new UuidRegistry(['table_name' => self::COVERAGE_TABLE]))->createMissingUuids();
```

Reply:
and then really no need for the `private $uuidRegistry` variable

Reply:
Done